template/express-rest-api/docker-compose.yml

```diff
@@ -2,16 +2,6 @@ version: '3.7'
 
 services:
   app:
-    environment:
-      # Enable Buildkite + GitHub integrations.
-      - BUILDKITE
-      - BUILDKITE_AGENT_ACCESS_TOKEN
-      - BUILDKITE_BRANCH
-      - BUILDKITE_BUILD_NUMBER
-      - BUILDKITE_JOB_ID
-      - BUILDKITE_PIPELINE_DEFAULT_BRANCH
-      - BUILDKITE_STEP_ID
-      - GITHUB_API_TOKEN
```

Comment:
`GITHUB_API_TOKEN` isn't a built-in Buildkite environment variable, so it may still need to be specified unless your testing shows otherwise:

https://github.com/search?q=org%3ASEEK-Jobs%20propagate-environment%20GITHUB_API_TOKEN&type=code

Reply:
![image](https://github.com/seek-oss/skuba/assets/18017094/cad02cb6-c0f6-4c88-ade8-91dbe4f9e5bd)

Looks like it fails

Reply:
I've added this to the pipeline environments so that it flows through to docker as well, just like the examples from this search: https://github.com/search?q=org%3ASEEK-Jobs%20propagate-environment%20GITHUB_API_TOKEN&type=code

Reply:
☝️ think defining it on the pipeline with the `propagate-environment` should fix this and we won't need to still pass the environment explicitly in the `docker-compose` anymore.

Reply:
Nice one, I like the symmetry between our `docker` and `docker-compose` usage 👍